### ISSUE TITLE
RepEnrich2_subset.py rna-seq PE error

### ISSUE THREAD

**Issue by Runuply** [nerettilab/RepEnrich2#16]
Hi Nick

RepEnrich2 works well when I am using the SE rna-seq data, however, if I use the PE, the RepEnrich2_subset.py script is crashed. Could you please let me know the solution?

Thanks,
<img width="1172" alt="Screen Shot 2019-07-03 at 2 48 08 PM" src="https://user-images.githubusercontent.com/18191776/60760613-0ecbdc80-a006-11e9-978e-52bd9c292c0a.png">
<img width="1164" alt="Screen Shot 2019-07-03 at 3 08 59 PM" src="https://user-images.githubusercontent.com/18191776/60760614-112e3680-a006-11e9-9795-7e979a1672ed.png">

**Comment by nskvir** [comment]
Is there any other output, or just the symbols above?  If the script isn't throwing a specific error message it's hard to discern what the problem is without being able to reproduce the issue on my end.

**Comment by Runuply** [comment]
There are outputs of filtered.bam and unique.bam, except for the multimap.fastq.
There is no specific error but only the running symbols. However, the single end data work well without any issue from beginning to the end.

![Screen Shot 2019-07-08 at 3 07 54 PM](https://user-images.githubusercontent.com/18191776/60836082-cc83d600-a192-11e9-9028-bf699c134a8d.png)

**Comment by nskvir** [comment]
It looks like it was in the process of sorting the bam file when the error occurred (although there should be a print statement that happens before that, which I do not see in your first screenshot).  Can you tell me what version of samtools you have installed?

**Comment by Runuply** [comment]
samtools 1.2
Using htslib 1.2.1
Copyright (C) 2015 Genome Research Ltd.

**Comment by nskvir** [comment]
I would start by trying a newer version of Samtools (v1.3.1 was what I used when I wrote the tutorial) in case there is some functionality that differs in the older versions.  

Also be sure the other versions of dependencies are all up to date as well -- versions of all the software I use are as follows: Python version 2.7.12+, Bedtools version 2.25.0, Bowtie2 version 2.2.9, Samtools version 1.3.1, and BioPython version 1.66-py2.7.  

Let me know if this helps!
